nexus/db-queries/src/db/datastore/volume.rs

```diff
@@ -3265,11 +3363,11 @@ pub fn count_read_write_sub_volumes(vcr: &VolumeConstructionRequest) -> usize {
             // We don't support a pure Region VCR at the volume
             // level in the database, so this choice should
             // never be encountered.
-            panic!("Region not supported as a top level volume");
+            bail!("Region not supported as a top level volume");
```

Comment:
These functions had an undocumented panic and instead of documenting it, it should return a 500. Nexus shouldn't panic here, as a badly constructed volume could be accidentally created and using it would always result in a panic.